index.html

```diff
@@ -5,12 +5,34 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+    <meta name="description" content="Portfolio website">
+    <meta name="keywords" content="HTML, CSS, Javascript">
+    <meta name="author" content="Maen Kh">
+    <link rel="stylesheet" href="/CSS/style.css">
     <title>Portfolio</title>
 </head>
 
 <body>
-
+    <!-- popup modal -->
+    <section class="modal-container">
+        <img class="modal-btn-close" src="/assets/images/modal-mobile-close.png" alt="">
+        <img class="modal-image" src="/assets/images/modal-mobile-img.png" alt="">
+        <div class="modal-title-btn-container">
+            <h1 class="modal-title">Keeping track of hundreds of components</h1>
+            <div class="modal-button">
+                <div class="btn btn-green"><a href="#">See Live <img class="icon-inside-button" src="/assets/images/modal-icon1.png" alt=""></a></div>
+                <div class="btn btn-green"><a href="#">See Source <img class="icon-inside-button" src="/assets/images/modal-icon2.png" alt=""></a></div>
+            </div>
```

Comment:
- [ ] Kindly your buttons are too big compared with the Figma design. Kindly make them smaller.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/156625211-2c5a2df9-e067-4dd6-baec-21d3ef3af565.png) | ![image](https://user-images.githubusercontent.com/35535130/156625287-4b9b5349-5fae-488f-8cfe-933091de1a3b.png) |

Reply:
Good job implementing this change 💪🏿 

- [ ] Just as the previous reviewer requested, kindly make the buttons smaller by reducing the width, so it looks like the original design 


 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/69638013/156876265-b92708e9-a528-418d-88a9-f673ac230d17.png) | ![Capture](https://user-images.githubusercontent.com/69638013/156876142-b1784106-f21e-49e7-9636-4a5791ca67d4.PNG)